components/salesforce_rest_api/actions/create-attachment/create-attachment.mjs

```diff
@@ -0,0 +1,60 @@
+import salesforce from "../../salesforce_rest_api.app.mjs";
+import attachment from "../../common/sobjects/attachment.mjs";
+import lodash from "lodash";
+import { toSingleLineString } from "../../common/utils.mjs";
+
+export default {
+  key: "salesforce_rest_api-create-attachment",
+  name: "Create Attachment",
+  description: toSingleLineString(`
+    Creates an attachment, which represents a file that a User has uploaded and attached to a parent object.
+    See [Attachment SObject](https://developer.salesforce.com/docs/atlas.en-us.228.0.object_reference.meta/object_reference/sforce_api_objects_attachment.htm)
+    and [Create Record](https://developer.salesforce.com/docs/atlas.en-us.228.0.api_rest.meta/api_rest/dome_sobject_create.htm)
+  `),
+  version: "0.3.2",
+  type: "action",
+  props: {
+    salesforce,
+    Body: {
```

Comment:
Some specific reason to have these props with the first letter in uppercase? I saw this behavior in many files. I am wondering if we can use `camelCase` for every prop